README.md

```diff
@@ -162,6 +162,75 @@ This will deploy a new Ingress Nginx Controller with TCP and HTTP Ingress
 ### Package
 1.) Package charts
 
-Package all charts, and created an index.yaml in ./repo directory
+<<<<<<< HEAD
```

Comment:
Looks like you had a bad merge. Please clean this up. I suspect this removed line 165.